exercises/Ex1AreaOfTriangle.js

```diff
@@ -0,0 +1,6 @@
+function triangleArea(a, b, c){
+    var halfTrianglePerimeter = (a+b+c)/2;
+    var triangleArea = Math.sqrt(halfTrianglePerimeter*(halfTrianglePerimeter-a)*(halfTrianglePerimeter-b)*(halfTrianglePerimeter-c));
+    return (console.log("The area of a triangle with sides " + a + ", " + b +" and " + c + " is " + triangleArea + "."));
```

Comment:
Your method needs to return the result (in this case - area of a triangle).
Putting executable stuff inside return (), you are returning the result of console.log execution (no idea what is it).